src/main/kotlin/com/progreizh/animaliste/repositories/AccountRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.progreizh.animaliste.repositories
+
+import com.progreizh.animaliste.entities.Account
+import org.springframework.data.mongodb.repository.MongoRepository
+import java.util.Optional
+
+interface AccountRepository : MongoRepository<Account, String> {
+    fun findAccountByMailEquals(mail: String): Optional<Account>
```

Comment:
```suggestion
    fun findAccountByMail(mail: String): Optional<Account>
```
Le nom findAccountByMailEquals est super bizare